pex/wrapper/pex_wrapper.py

```diff
@@ -109,6 +110,10 @@ def main():
                 raise RuntimeError("Failed to add %s: %s" % (egg, err))
 
         # TODO(mikekap): Do something about manifest['nativeLibraries'].
```

Comment:
Remove the TODO comment please, since you have now Done Something about nativeLibraries :-)

Reply:
Done